projects/documentation/scripts/build-search-index.js

```diff
@@ -18,7 +18,12 @@ import { fileURLToPath } from 'url';
 const __dirname = path.dirname(fileURLToPath(import.meta.url));
 
 const projectDir = path.resolve(__dirname, '..', '..', '..');
-const indexDir = path.resolve(projectDir, 'projects/documentation/dist');
+const indexDir = path.resolve(
+    projectDir,
+    process.env.SWC_DIR
+        ? 'projects/documentation/dist'
+        : 'projects/documentation/_site/src/'
+);
```

Comment:
Thank you!

Reply:
A little wonky yet in the preview environment. Trying a little different approach.